lib/hammer_cli_foreman/auth_source_external.rb

```diff
@@ -0,0 +1,30 @@
+module HammerCLIForeman
+  class AuthSourceExternal < HammerCLIForeman::Command
+    resource :auth_source_externals
+    command_name 'external'
+    desc _('Manage external auth sources')
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _('Id')
+        field :name, _('Name')
+        field :location_id, _('Location')
+        field :organization_id, _('Organization')
```

Comment:
@ofedoren while I am testing the PR, I am unable to get the location and organization values in the list command. Can you suggest a way in which users can see the same?

Reply:
Or I would require the info command to show the same?

Reply:
Yes, I've noticed that too. This is because the response from the server contains `organizations/locations` collections instead of single org/loc. So the best way here is to change those two fields to `HammerCLIForeman::References.taxonomies(self)`. BUT this will not work either for `table` output adapter because taxonomies fields will be cleared from output fields because of hammer core design (since collections contain more than one piece of data for a field, we don't show them in tables because output can be broken).

This is what I'm suggesting:
  1. Since external auth source can have multiple org/loc, change two fields as I mentioned, so the taxonomies are shown at least if `--output=base` is used.
  2. Change the default adapter to be `base`, so the taxonomies are shown always (but this will not be consistent with other list commands' output).
  3. OR instead of 1 and 2 implement `List` command without taxonomies and implement `Info` command with taxonomies.

I'd prefer the 3'd way, because it will be consistent with other commands and also we implement `Info` command for the endpoint we have.

Reply:
@ofedoren: myself and @shiramax were also talking about it and figured out the same thing as you suggested. Looks like we are all on the same page on this! 

Will update the PR with all the changes and add test cases to it :) thank you both.